### PR TITLE
Adding T1530 Test 1 - Enumerate Azure Blobs with MicroBurst

### PR DESCRIPTION
**Pull request by Leomon5**
**Details:**
Creating T1530 Test 1 - Enumerate Azure Blobs with MicroBurst. This test uses Netspi's MicroBurst Powershell module to enumerate public-facing Azure storage containers/blobs. To run the test, users must specify a base keyword to start with (typically the storage account name is a good place to start, since every Azure storage account should create a url of [storageaccountname].blob.core.windows.net), which is then combined with a wordlist that creates search permutations based on common sensitive file names. See https://www.netspi.com/blog/technical/cloud-penetration-testing/anonymously-enumerating-azure-file-resources/ . 

**Testing:**
Tested on Windows 10 with a personal public-facing Azure storage account/public container

T1530 Azure Blob Enumeration - Redacted

### CODE REVIEW

File: atomics/T1530/T1530.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+attack_technique: T1530
+display_name: Data from Cloud Storage Object
+atomic_tests:
+- name: Enumerate Azure Blobs with MicroBurst
```
Comment (patel-bhavin) [inline atomics/T1530/T1530.yaml:4]: Can you update the name to Azure - Enumerate Azure Blobs with MicroBurst

File: atomics/T1530/T1530.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  description: |
+    Upon successful execution, this test will utilize a wordlist to enumerate the public facing containers and blobs of a specified Azure storage account. 
+    See https://www.netspi.com/blog/technical/cloud-penetration-testing/anonymously-enumerating-azure-file-resources/ . 
+  supported_platforms:
+  - windows
```
Comment (patel-bhavin) [inline atomics/T1530/T1530.yaml:9]: Change windows to iaas:azure

**Comment by patel-bhavin** [comment]
@Leomon5 : i have added inline comments to this PR. Thanks!

**Comment by Leomon5** [comment]
Hi @patel-bhavin , I've made the two changes as requested. Please let me know if you have any other suggestions! Thank you.

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**